proselint/checks/cursing/nfl.py

```diff
@@ -12,16 +12,18 @@
 Words the NFL won't print on a jersey.
```

Comment:
While humorous, I was never really a fan of these checks. I am flagging this here to create a forum for discussion of the check restructuring and recategorising, since a pull request of this size seems like the perfect opportunity.

Reply:
as an optional check ok - I at least learned some new connotations :)
these checks are not harmful and have a very low false-positive rate. my latest dev-branch has fixes for them as they used things like `a.s.s.` which has "." as unwanted active regex-component.

Reply:
My point is more that these specifically feel unnecessary. When people use these words, they're used intentionally. Trying to catch every variant of a curse word is computationally expensive, and I don't really feel it should be proselint's job.

Reply:
cursing-checks are not even in the top10 of computational expensive, so I'm not concerned. But we can disable them by default. I consider most other checks the same as you see the cursing part. You either break the rules intentionally or you don't know about it and get educated by the linter and can decide for yourself.